app/v1/models/red_flag_model.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+import uuid
+
+red_flag_data = []
+class RedFlagModel(object):
+    """Red-flag domain model"""
+
+    def __init__(self):
+        self.red_flag_data = red_flag_data
+
+    def save(self, red_flag_dict):
+        """Append the data in the red-flag dictionary into a red-flag list"""
+        
+        payload = {
+            'id': uuid.uuid4(),
+            'created_on': datetime.datetime.now(),
+            'created_by': red_flag_dict['created_by'],
+            'record_type': red_flag_dict['record_type'],
+            'location': red_flag_dict['location'],
+            'status': red_flag_dict['status'],
+            'image': red_flag_dict['image'],
+            'video': red_flag_dict['video'],
+            'comment': red_flag_dict['comment']
+        }
+        self.red_flag_data.append(payload)
+        return self.red_flag_data
+
+    
+    def get_red_flags(self):
+        """Return all created red-flags"""
+        if len(self.red_flag_data) < 0:
```

Comment:
These checks should be done in the views

Reply:
Sure. I will factor that out.